src/validations/profileSchema.js

```diff
@@ -0,0 +1,95 @@
+
+const profileSchemas = {
+  profileUpdateSchema: [
+    check('userName')
+      .optional()
+      .trim()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('userName cannot be blank')
+      .isLength({ min: 2, max: 20 })
+      .withMessage('username must be at least 2 characters, and maximum 20')
+      .not()
+      .matches(/^[A-Za-z]+[-]{1}[A-Za-z]+([-]{1}[A-Za-z]+)+$/, 'g')
+      .withMessage('invalid input for username')
+      .not()
+      .matches(/^[A-Za-z]+[']+[A-Za-z]+[']+[A-Za-z]+$/, 'g')
+      .withMessage('invalid input for username')
+      .matches(
+        /^[A-Za-z]+(['-]?[A-Za-z]+)?([ -]?[A-Za-z]+)?(['-]?[A-Za-z]+)?$/,
+        'g'
+      )
+      .withMessage('invalid input for username')
+      .customSanitizer(value => makeUpperCase(value)),
+    check('firstName')
+      .optional()
+      .trim()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage(' firstName cannot be blank')
+      .isLength({ min: 2, max: 20 })
+      .withMessage('firstName must be at least 2 characters, and maximum 20')
+      .not()
+      .matches(/^[A-Za-z]+[-]{1}[A-Za-z]+([-]{1}[A-Za-z]+)+$/, 'g')
+      .withMessage('invalid input for firstName')
+      .not()
+      .matches(/^[A-Za-z]+[']+[A-Za-z]+[']+[A-Za-z]+$/, 'g')
+      .withMessage('invalid input for firstName')
+      .matches(
+        /^[A-Za-z]+(['-]?[A-Za-z]+)?([ -]?[A-Za-z]+)?(['-]?[A-Za-z]+)?$/,
+        'g'
+      )
+      .withMessage('invalid input for firstName')
+      .customSanitizer(value => makeUpperCase(value)),
+    check('lastName')
+      .optional()
+      .trim()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('lastName cannot be blank')
+      .isLength({ min: 2, max: 20 })
+      .withMessage('lastName must be at least 2 characters, and maximum 20')
+      .not()
+      .matches(/^[A-Za-z]+[-]{1}[A-Za-z]+([-]{1}[A-Za-z]+)+$/, 'g')
+      .withMessage('invalid input for lastName')
+      .not()
+      .matches(/^[A-Za-z]+[']+[A-Za-z]+[']+[A-Za-z]+$/, 'g')
+      .withMessage('invalid input for lastName')
+      .matches(
+        /^[A-Za-z]+(['-]?[A-Za-z]+)?([ -]?[A-Za-z]+)?(['-]?[A-Za-z]+)?$/,
+        'g'
+      )
+      .withMessage('invalid input for lastName')
+      .customSanitizer(value => makeUpperCase(value)),
+    check('email')
+      .optional()
+      .trim()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('email cannot be blank')
+      .isEmail()
+      .withMessage('email is not valid')
+      .customSanitizer(value => value.toLowerCase()),
+    check('avatarUrl')
+      .optional()
+      .not()
+      .isEmpty({ ignore_whitespace: true })
+      .withMessage('avatarUrl cannot be blank')
+      .isURL()
+      .withMessage('avatarUrl must be a valid URL string'),
+    check('id')
+      .trim()
+      .isUUID()
+      .withMessage('invalid id')
+  ],
+  profileViewSchema: [
```

Comment:
Leave a new live between these sort of object properties so that the code is easy to read

Reply:
Done

Reply:
For some reason, I can't see that this is done. I expect to see a new line between lines 56  and 57. Kindly recheck